### PR TITLE
xyz@1.0.x

### PR DESCRIPTION
**Pull request by davidchambers**
davidchambers/xyz#22

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "ramda",
   "main": "dist/ramda.js",
-  "version": "0.22.1",
```
Comment (davidchambers) [inline bower.json]: davidchambers/xyz#19

**Comment by buzzdecafe** [comment]
:cow2:

**Comment by buzzdecafe** [comment]
:dromedary_camel:

**Comment by davidchambers** [comment]
Anyone who publishes Ramda should run `rm -r node_modules && npm install`. :)

**Comment by buzzdecafe** [comment]
@davidchambers would it be overkill to add that to the publish flow?

**Comment by davidchambers** [comment]
Something like this, do you mean?

```diff
 .PHONY: release-major release-minor release-patch
 release-major release-minor release-patch:
+	@rm -rf node_modules
+	@npm install
 	@$(XYZ) --increment $(@:release-%=%)
```

We could certainly do so, though publishing would become slightly less satisfying. I'm don't mind one way or the other.

**Comment by buzzdecafe** [comment]
yeah that's what i mean. i like it because it's one less thing to remember.

**Comment by davidchambers** [comment]
`xyz@1.0.0` is the first release since November 2014. It may well be another two years before the next release. So running `rm -r node_modules && npm install` is a once-every-year-or-two task rather than a run-before-publishing task. I would not even have mentioned it but for the change which means the `"version"` field in __bower.json__ is no longer updated. Running an older version of xyz would reintroduce the `"version"` field removed in this pull request.

**Comment by buzzdecafe** [comment]
i see. nm then :-)

**Comment by svozza** [comment]
Would `npm prune` achieve the same thing?

**Comment by davidchambers** [comment]
> Would `npm prune` achieve the same thing?

I don't think so. Its [documentation][1] says nothing about updating dependencies.


[1]: https://docs.npmjs.com/cli/prune

**Comment by svozza** [comment]
I meant in conjunction with `npm install `.

**Comment by davidchambers** [comment]
I don't think `npm install` updates dependencies either. [`npm update`][1] does, but I've had problems with that command in the past. npm is a complex, bug-ridden piece of software. I rely on it as little as possible. ;)


[1]: https://docs.npmjs.com/cli/update

**Comment by svozza** [comment]
Haha. Fair enough.

**Pull request merged**